docker-compose.yml

```diff
@@ -411,6 +411,25 @@ services:
     depends_on:
       - export-client
 
+  system:
+    image: ${systemManagement}
+    ports:
+      - "48090:48090"
+    container_name: edgex-sys-mgmt-agent
+    hostname: edgex-sys-mgmt-agent
+    networks:
+      - edgex-network
+    environment:
+      <<: *common-variables
+    volumes:
+      - db-data:/data/db
+      - log-data:/edgex/logs
+      - consul-config:/consul/config
+      - consul-data:/consul/data
+      - /var/run/docker.sock:/var/run/docker.sock
```

Comment:
Can we add a comment describing what this volume is used for so that we are not scratching our heads in the future? The only reason I suggest that is this is a one off from the other services, and at first glance I would be wondering if this is needed and if so why.